contracts/DCAHubCompanion/DCAHubCompanionTakeWithdrawAndSwapHandler.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@mean-finance/swappers/solidity/contracts/SwapAdapter.sol';
+import '../interfaces/IDCAHubCompanion.sol';
+
+/// @dev All public functions are payable, so that they can be multicalled together with other payable functions when msg.value > 0
+abstract contract DCAHubCompanionTakeWithdrawAndSwapHandler is SwapAdapter, IDCAHubCompanionTakeWithdrawAndSwapHandler {
```

Comment:
In a following PR, I'll Swap Extensions to the contract

Reply:
I'm assuming you will add the extensions? Hehe